magento/magento2
# Wrong currency sign on creditmemo_grid & sales_order_view > invoice grid

### Description (*)

Added missing column `base_currency_code` & `order_currency_code` to table `sales_creditmemo_grid`. Price column rendering changed on `sales_creditmemo_index` (Creditmemo grid) & `sales_order_view` - Invoice tab.

### Fixed Issues
1. magento/magento2#22490

### Manual testing scenarios (*)

1. Create order in different currency as configured on default store view
2. Go to Backend > SALES > Credit Memos, create Credit Memo
3. Check Refunded & Subtotal column on Credit Memo grid (`sales_creditmemo_index`). You should see price in currency configured on default store view (base_currency_code).
4. Go to Backend > SALES > Orders > choose your order and check _Invoice grid_ in Invoices tab.  **Amount** should be `base grand total` with `base_currency_code`.

sales_order_view > Credit Memos (base grand total):
![image](https://user-images.githubusercontent.com/5650354/55455850-9a883400-55e5-11e9-8e14-f497c5914d4e.png)

sales_order_view > Invoices (grand total):
![image](https://user-images.githubusercontent.com/5650354/55456082-5ea19e80-55e6-11e9-9401-d02757183016.png)

sales_creditmemo_index (wrong currency sign):
![image](https://user-images.githubusercontent.com/5650354/55456087-63665280-55e6-11e9-8b6c-2a547a1ac40f.png)


### Contribution checklist (*)
